opentelemetry-exporter-gcp-monitoring/tests/test_user_agent.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+
+from fixtures.gcmfake import GcmFake, GcmFakeMeterProvider
+
+
+def test_with_resource(
+    gcmfake_meter_provider: GcmFakeMeterProvider,
+    gcmfake: GcmFake,
+) -> None:
+    meter_provider = gcmfake_meter_provider()
+    counter = meter_provider.get_meter(__name__).create_counter(
+        "mycounter", description="foo", unit="{myunit}"
+    )
+    counter.add(12)
+    meter_provider.force_flush()
+
+    for calls in gcmfake.get_calls().values():
+        for call in calls:
+            assert (
+                re.match(
+                    r"^opentelemetry-python \S+; google-cloud-metric-exporter \S+ grpc-python/\S+",
```

Comment:
@punya does this look good to you? I think the collector exporter just sets `opentelemetry-collector-contrib` but this seems a little more useful.

Lmk if you prefer a different format

Reply:
This is what we do for go: https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/blob/fbfa6aa3082c07d81ae7ad61e8ccdea9c85b63f6/exporter/metric/option.go#L29

`opentelemetry-go %s; google-cloud-metric-exporter %s`